src/PokeUsers.class.js

```diff
@@ -0,0 +1,14 @@
+const PokeUser = require('./PokeUser.class')
```

Comment:
Didn't get what you want ;)
I created a `PokeUsers` to store in-memory all users (by UUID).
express-session create on-th-fly a user if not found.